src/ui/page/notFoundError/notFoundError.tsx

```diff
@@ -0,0 +1,50 @@
+import type { FC } from 'react'
+import { useCallback } from 'react'
+import { useNavigate } from 'react-router-dom'
+import classNames from 'classnames'
+import { useTranslation } from 'react-i18next'
+import { useAppStore } from '@/ui/store/appStore'
+import { Button } from '@/ui/component/button/button'
+import '@/ui/page/notFoundError/i18n/index'
+import '@/ui/page/home/i18n/index'
+import './notFoundError.scss'
+
+export const NotFoundError: FC = () => {
+  const { t } = useTranslation(['notFoundError', 'home'])
+  const navigate = useNavigate()
+  const theme = useAppStore(store => store.theme)
+
+  const backToDataverse = useCallback((): void => navigate('/dataverse'), [navigate])
+  const backToHome = useCallback((): void => navigate('/'), [navigate])
+
```

Comment:
You can refactor this piece of code by having a single function that handle navigation for the whole page 😌

Reply:
True, done!